devenv/README.md

```diff
@@ -57,13 +57,11 @@ Jaeger block runs both Jaeger and Loki container. Loki container sends traces to
 
 ## Troubleshooting
 
-### Containers fail to start (Mac OS)
+### Containers that read from log files fail to start (Mac OS)
+
+If you are running Mac OSX, containers that read from the log files (e.g. telegraf, fileabeat, promtail) sometimes fail to start. This is because default Docker for Mac does not have permission to create `grafana` folder at `var/log` location, as it runs as the current user. To solve this issue, manually create the folder `/var/log/grafana`, then try starting the containers again.
```

Comment:
```suggestion
If you are running Mac OSX, containers that read from the log files (e.g. telegraf, fileabeat, promtail) sometimes fail to start. This is because default Docker for Mac does not have permission to create `grafana` folder at `/var/log` location, as it runs as the current user. To solve this issue, manually create the folder `/var/log/grafana`, then try starting the containers again.
```